src/System.Threading/tests/BarrierTests.cs

```diff
@@ -263,7 +263,7 @@ public static void RunBarrierTest8_PostPhaseException()
                     if (shouldThrow)
                         throw new InvalidOperationException();
                 });
-            int succeededCount = 0;
+            long succeededCount = 0;
```

Comment:
Correct, it was changed to use the ```long``` version of Interlocked.Increment and Interlocked.Decrement since we use the ```int``` version in all other test. 

I'll give them their own test.